src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -283,6 +285,64 @@ def update_app_service_plan(instance, sku=None, number_of_workers=None,
         instance.admin_site_name = admin_site_name
     return instance
 
+def show_backup_configuration(resource_group_name, name, slot=None):
+    return _generic_site_operation(resource_group_name, name, 'get_site_backup_configuration',
+                                   slot)
+
+def list_backups(resource_group_name, name, slot=None):
+    return _generic_site_operation(resource_group_name, name, 'list_site_backups',
+                                   slot)
+
+def create_backup(resource_group_name, name, storage_account_url,
+                  database_name=None, database_type=None,
+                  database_connection_string=None, backup_name=None, slot=None):
+    client = web_client_factory()
+    location = _get_location_from_webapp(client, resource_group_name, name)
+    db_setting = DatabaseBackupSetting(database_type, database_name,
+                                       connection_string=database_connection_string)
```

Comment:
For create backup, would the database parameters not be required? Also, we don't recommend you use "name" as a parameter argument because it isn't clear what it refers to. Is it webapp_name?